src/main/java/pingis/controllers/TaskController.java

```diff
@@ -19,7 +19,7 @@
 import pingis.entities.ImplementationType;
 import pingis.entities.TaskImplementation;
 
-import pingis.entities.TmcSubmission;
+import pingis.entities.tmc.TmcSubmission;
```

Comment:
I left it as is for now as the Tmc prefix is pretty prevalent and I wasn't sure if TmcSubmissionController etc. should be changed as well. Would be a good idea for a separate PR, I think.